src/main/java/teammates/ui/request/ReadNotificationCreateRequest.java

```diff
@@ -0,0 +1,30 @@
+package teammates.ui.request;
+
+/**
+ * The request of creating a read notification in account.
+ */
+public class ReadNotificationCreateRequest extends BasicRequest {
```

Comment:
Can consider renaming this to `MarkNotificationAsReadRequest`, since the current naming may cause confusion with `NotificationCreateRequest`.